SparkPost/2web2ui
# AC-770: Recipient validation pricing

### What Changed
 - Added recipient validation to billing summary

### How To Test
 - On an account w/ recipient validation enabled, navigate to billing summary page
- See that usage calculates price
 - Click on `How is this calculated?`
 - See that usage calculation is broken down by tier

### To Do
- [x] Consolidate load conditions
- [x] Remove redundant price calculation (?)
- [x] Add testing for modal content
- [x] Check code coverage
